Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "coreos-metadata"
+version = "0.1.0"
```

Comment:
So it's mentioned here and not just in the overlay PR, this should also be updated to `1.0.0`.